datalad/distribution/dataset.py

```diff
@@ -223,6 +223,27 @@ def get_subdatasets(self, pattern=None, fulfilled=None, absolute=False,
 #        """
 #        raise NotImplementedError("TODO")
 
+    # TODO maybe needs to get its own interface
+    def remember_state(self, message, auto_add_changes=True, version=None):
```

Comment:
Actually not part of this PR. Replaced by `save` as you mentioned. (remember: toplevel functions are methods of `Dataset`). So, it's a matter of merge. Should disappear completely.